docker-compose.web.yml

```
@@ -0,0 +1,65 @@
+    # sudo chown -R $(id -u):$(id -g) ./code
+    # Then change to: user: "${UID:-1000}:${GID:-1000}"
+    user: "0:0"
+    environment:
+      - PYTHONUNBUFFERED=1
+      # Use /tmp as HOME since /root may not be writable
+      - HOME=/tmp
+      # Git config for worktree operations
+      - GIT_DISCOVERY_ACROSS_FILESYSTEM=1
+    env_file:
+      - .env.docker
+    environment:
```

YAMLlint (1.38.0)

[error] duplication of key "environment" in mapping

(key-duplicates)